ethereum/spec/src/main/java/tech/pegasys/teku/spec/Spec.java

```diff
@@ -197,11 +196,24 @@ public SpecConfig getGenesisSpecConfig() {
     return getGenesisSpec().getConfig();
   }
 
+  /**
+   * Base networking constants
+   *
+   * <p>These constants are unified among forks and are not overriden, new constant name is used if
+   * it's changed in the new fork
+   */
   public NetworkingSpecConfig getNetworkingConfig() {
     // Networking config is constant along forks
     return getGenesisSpec().getConfig().getNetworkingConfig();
   }
 
+  /** Networking config with Deneb constants */
+  public Optional<NetworkingSpecConfigDeneb> getNetworkingConfigDeneb() {
```

Comment:
In my opinion if we always do `orElseThrow()` when using this we better make it not an optional, we can have `NetworkingSpecConfigDeneb.required()` in similar pattern we use for the SpecConfig and just throw exception if it's called.